locust-metrics/README.md

```diff
@@ -1,3 +1,47 @@
 # Locust Metrics Example
 
-This example shows how to optimize the voting app using the metrics built into Locust. This is useful when tuning an application the does not have metrics yet. Locust is now run as a deployment that lasts the length of the experiment rather than as the trial pod. The trial pod interacts with the Locust API to start and stop load tests.
+This example shows how to optimize the voting app using:
+- the experiment generator
+- the metrics built into Locust
+
+This is useful when tuning an application that does not have metrics yet.
+
+## Deploy the voting app
+
+Run
+```
+kustomize build . | kubectl apply -f -
+```
+## Create an experiment
+
+The `app.yaml` file defines the experiment to be generated. In this experiment we use Locust for the load test
+
+```
+scenarios:
+- name: 100-clients
+  locust:
+    users: 400
+    spawnRate: 20
+    locustfile: locustfile.py
+```
+
+We optimize for cost and p95 latency as provided by Locust:
+```
+objectives:
+- name: cost-gcp # Also "cost", "cost-aws", "cpu", ...
+- name: p95-latency
+  latency: p95
+```
+
+Generate the `experiment.yaml` file using
+```
+redskyctl generate experiment -f app.yaml > experiment.yaml
+```
+Generate the RBAC permission that allow the controller to patch the deployment:
+```
+redskyctl generate rbac -f experiment.yaml
```

Comment:
Wouldn't this need to go out to file as well?

Reply:
forgot to apply the file generated, thanks for catching that.